tests/unit/Mvc/Collection/AggregateCest.php

```diff
@@ -1,31 +0,0 @@
-<?php
```

Comment:
Are these deleted in anticipation of the new mongo collection object?

Reply:
All of the cests deleted from unit/Mvc/ also exist in integration/. As they require the DI so they should be in integration/ so these have been deleted.